### DISCUSSION TITLE
I'm Interested in the next Cohort!

### DISCUSSION THREAD

**Discussion by keanyaphelps** [djangonaut-space/program#17] [Q&A] [answered]
Hello Djangonauts!

Please keep me on the radar for the next Cohort.  I'm so very interested in this wonderful program!

Keanya

**Comment by RachellCalhoun** [comment, marked as answer]
Hi Keanya and thanks for the interest in the program! We are still finalizing the details for the next program session, like the dates and application process, but if you'd like to get an email when it's ready to go please fill out this google form: https://forms.gle/4vNuqQwEK6o48WqHA and we'll reach out again before we start our next session. Hope to hear from you soon :)
